src/docs/ftw-introduction/introducing-ftw-x/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Sharetribe Web Template
+slug: sharetribe-web-template
+updated: 2023-01-01
+category: ftw-introduction
+ingress:
+  This article introduces the Sharetribe Web Template and links to
+  relevant resources.
+published: true
+---
+
+## Introduction
+
+[The Sharetribe Web Template](https://github.com/sharetribe/ftw-x) is a
+React template meant to function as a starting point for the development
+of your marketplace. It supports all the features available through the
+Sharetribe APIs, and is easily customized. For a complete list of
+features supported by the template, see the
```

Comment:
This wording might be interpreted so that the template natively supports everything in the features list (e.g. discount codes, custom cancellation policies, multiple languages,...). So I'd suggest a wording to the tone of "For a complete list of features that are possible in the template, either natively or with custom development, see..." or similar.